soccer/gameplay/robocup-py.cpp

```diff
@@ -646,6 +650,7 @@ BOOST_PYTHON_MODULE(robocup) {
         .def("__repr__", &Point_repr)
         .def("normalized", &Geometry2d::Point::normalized, Point_overloads())
         .def("rotate", &Point_rotate)
+        .def("rotate_origin", &Point_rotate_origin)
```

Comment:
I think since `Point_rotate_origin` is a single line, this might be able to just be `&Geometry2d::Point::angle` maybe? I haven't tested it though.

Reply:
Unless I am misunderstanding your question I think that you have to make a wrapper for any of the functions that directly alter the point that it is passed through pointers since python doesn't seem to directly support that.

Reply:
ah, that's right, I forgot about that, this seems good then to me!